fastlane/metadata/tr/description.txt

```diff
@@ -1,11 +0,0 @@
-WordPress blogunuzu veya web sitenizi doğrudan iOS cihazınızdan yönetin veya oluşturun: Gönderiler ve sayfalar oluşturun ve düzenleyin, en sevdiğiniz fotoğraf ve videoları yükleyin, istatistikleri görüntüleyin ve yorumları yanıtlayın.
```

Comment:
🤔  429 on `tr`?

Didn't you get the `retry` prompt (from https://github.com/wordpress-mobile/release-toolkit/pull/402)? Or maybe you haven't update to latest `release-toolkit` yet? 🤔  (And if so, would probably be worth to update it in the release branch before the final build tomorrow)

Reply:
For posterity, this has been addressed in the description of https://github.com/wordpress-mobile/WordPress-iOS/pull/19275